tools/dom-test-helpers.js

```diff
@@ -1,6 +1,6 @@
 import { getComposedChildren } from '../helpers/dom.js';
 
-export function keyDown(element, keyCode, ctrlKey) {
+export function keyDown(element, keyCode) {
```

Comment:
None of the remaining uses set this, so figured we could at least clean that up now